app/src/main/java/nat/chung/mediadecoderplayer/DecodePlayer.java

```diff
@@ -38,7 +38,8 @@
     enum PLAY_TASK_STATUS {
         PLAY_TASK_RUNNING,
         PLAY_TASK_STOPING,
-        PLAY_TASK_STOPPED
+        PLAY_TASK_STOPPED,
+        PLAY_TASK_PAUSE
```

Comment:
IF you added a new status, you should check all of the statuses that how it use.